src/main/windows/main.ts

```diff
@@ -262,6 +262,11 @@ class Main extends Route {
             accelerator: 'CommandOrControl+Alt+F',
             click: () => this.win.webContents.send ( 'window-focus-toggle' )
           },
+          {
+            label: 'Toggle Statusbar',
+            accelerator: 'CommandOrControl+Alt+.',
```

Comment:
This shortcuts sounds a bit weird to me. I would propose `CommandOrControl+Alt+S` but I'd rather use that for toggling split-view. 

Maybe it's ok to not assign a shortcut to this action as most people would rather need this or don't care about it?

Reply:
That's actually pretty common in editors, which is how I knew about it in the first place.

Reply:
I can't remember off the top of my head, I just know I've used it before. Regardless, I'm willing to change it to whatever shortcut you'd prefer. I do think that there should either be a shortcut key for it, or a way to persistently save whether its open or not between sessions. Otherwise it's a little annoying for people who _do_ want it to need to reopen it every time via the View menu.